CAFAna/Cuts/TruthCuts.h

```diff
@@ -62,21 +65,39 @@ namespace ana
             pos_z_cm < 1244);
   }
 
+
+  // Based just on PDG code and not on oscillation channel
+  const Cut kActuallyIsNumuCC([](const caf::StandardRecord *sr) {
```

Comment:
Let's change `kIsNumuCC` above to `kIsNumuCCSurv` or similar so these can drop the 'actually'